samples/knowledge_base_management.py

```diff
@@ -0,0 +1,151 @@
+  create --display-name DISPLAY_NAME
+  python knowledge_base_management.py --project-id PROJECT_ID \
+  get --knowledge-base-id knowledge_base_id
+  python knowledge_base_management.py --project-id PROJECT_ID \
+  delete --knowledge-base-id knowledge_base_id
+"""
+
+import argparse
+
+
+# [START dialogflow_list_knowledge_base]
+def list_knowledge_bases(project_id):
+    """Lists the Knowledge bases belonging to a project.
+
+    Args:
+        project_id: The GCP project linked with the agent."""
+    import dialogflow_v2beta1 as dialogflow
+    client = dialogflow.KnowledgeBasesClient()
+    agent_path = client.project_path(project_id)
+
+    print('Knowledge Bases for: {}'.format(project_id))
+    for knowledge_base in client.list_knowledge_bases(agent_path):
+        print(' - Display Name: {}'.format(knowledge_base.display_name))
+        print(' - Knowledge ID: {}\n'.format(knowledge_base.name))
+# [END dialogflow_list_knowledge_base]
+
+
+# [START dialogflow_create_knowledge_base]
+def create_knowledge_base(project_id, display_name):
+    """Creates a Knowledge base.
+
+    Args:
+        project_id: The GCP project linked with the agent.
+        display_name: The display name of the Knowledge base."""
+    import dialogflow_v2beta1 as dialogflow
+    client = dialogflow.KnowledgeBasesClient()
+    agent_path = client.project_path(project_id)
+
+    knowledge_base = dialogflow.types.KnowledgeBase(
+        display_name=display_name)
+
+    response = client.create_knowledge_base(agent_path, knowledge_base)
+
+    print('Knowledge Base created:\n')
+    print('Display Name: {}\n'.format(response.display_name))
+    print('Knowledge ID: {}\n'.format(response.name))
+# [END dialogflow_create_knowledge_base]
+
+
+# [START dialogflow_get_knowledge_base]
+def get_knowledge_base(project_id, knowledge_base_id):
+    """Gets a specific Knowledge base.
+
+    Args:
+        project_id: The GCP project linked with the agent.
+        knowledge_base_id: Id of the Knowledge base."""
+    import dialogflow_v2beta1 as dialogflow
+    client = dialogflow.KnowledgeBasesClient()
+    name = client.knowledge_base_path(project_id, knowledge_base_id)
+
+    response = client.get_knowledge_base(name)
+
+    print('Got Knowledge Base:')
+    print(' - Display Name: {}'.format(response.display_name))
+    print(' - Knowledge ID: {}'.format(response.name))
+# [END dialogflow_get_knowledge_base]
+
+
+# [START dialogflow_delete_knowledge_base]
+def delete_knowledge_base(project_id, knowledge_base_id):
+    """Deletes a specific Knowledge base.
+
+    Args:
+        project_id: The GCP project linked with the agent.
+        knowledge_base_id: Id of the Knowledge base."""
+    import dialogflow_v2beta1 as dialogflow
+    client = dialogflow.KnowledgeBasesClient()
+    name = client.knowledge_base_path(project_id, knowledge_base_id)
+
+    response = client.delete_knowledge_base(name)
+
+    print('Knowledge Base deleted.'.format(response))
```

Comment:
missing `{}` for formatting?  also here the `response` probably doesn't contain anything useful information.

Reply:
If I recall correctly this is to force a wait for the results. 
For tests to run properly.